tests/ieee802154_submac/Makefile

```diff
@@ -27,6 +27,8 @@ USEMODULE += ieee802154
 USEMODULE += ieee802154_submac
 USEMODULE += ztimer_usec
 
+include $(RIOTMAKE)/default-radio-settings.inc.mk
+
```

Comment:
Include it after ` include $(RIOTBASE)/Makefile.include` or RITOMAKE does not yet exist